### PR TITLE
adding a new addon role for Singularity

### PR DESCRIPTION
**Pull request by strus38**
Singularity is available as Opensource (aka. Singularity CE) or SingularityPRO (need a Sylabs subscription).
Singularity allows running jobs in containers.
More info: https://sylabs.io/singularity/

Thanks.

### CODE REVIEW

**Review by btravouillon — COMMENTED** [review]
Thank you for this PR. Can you answer my comment inline?

File: roles/addons/singularity/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Singularity CE or SingularityPRO Ansible role
```
Comment (btravouillon) [inline roles/addons/singularity/README.md:1]: I may rename the title to `Singularity`, which is the name of the role.

File: roles/addons/singularity/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## Description
+
+Install and configure Singularity CE or SingularityPRO
+
+## Pre-requisites
+
+* RPMs of Singulary CE
```
Comment (btravouillon) [inline roles/addons/singularity/README.md:9]: typo: Singular**it**y

File: roles/addons/singularity/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Note: RPM is not directly available on Singularity GitHub.
+RHEL8 build instructions (exemple done on management1, but can be done on another RHEL8 server** anywhere)
```
Comment (btravouillon) [inline roles/addons/singularity/README.md:11]: example

**Comment by strus38** [comment]
all fixed.

**Comment by oxedions** [comment]
Hello.

Same, thanks a lot for this PR. I will test it.

Ox

**Comment by strus38** [comment]
One thing I don't know: does the défaut opensource slurm has the option to
launch a singularity container... I hope so!

File: roles/addons/singularity/README.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
-# Singularity CE or SingularityPRO Ansible role
+# Singularity Ansible role
```
Comment (btravouillon) [inline roles/addons/singularity/README.md:1]: All the roles in the roles directory are Ansible roles. This title is used to generate [the online documentation](http://www.bluebanquise.com/documentation/roles.html#core), *Singularity* is enough.

Comment (strus38) [reply roles/addons/singularity/README.md]: done

**Review by btravouillon — APPROVED** [review]
Looks good to me. 👍

**Review by oxedions — APPROVED** [review]
Dear @strus38 ,

Thanks a lot for this impressive work.

Reviewed, it is ok for me. 😊 
Just not standard tasks naming, but not an issue for an addon, it is up to dev.

Merging.

**Pull request merged**